.github/workflows/repository-management.yml

```diff
@@ -0,0 +1,250 @@
+---
+name: Repository Management
```

Comment:
⛏️ I try to not treat most of these names and descriptions as Titles but statements with just one leading capitalization.

Reply:
I'll go ahead and update the name and then make a card for updating the linter with a rule for that.